__tests__/app.test.js

```diff
@@ -30,15 +29,43 @@ describe("GET /api/categories", () => {
   });
 });
 
+describe("GET /api/reviews/:review_id", () => {
+  test("200: should return a review object, with correct properties: ", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body: { review } }) => {
+        expect(review).toEqual({
+          review_id: 1,
+          title: "Agricola",
+          review_body: "Farmyard fun!",
+          designer: "Uwe Rosenberg",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          votes: 1,
+          category: "euro game",
+          owner: "mallionaire",
+          created_at: "2021-01-18T10:00:20.514Z",
+        });
+      });
+  });
+  test("404: not found when review_id does not exist", () => {
+    return request(app)
+      .get("/api/reviews/100")
+      .expect(404)
+      .then(({ body: {msg}}) => {
+        expect(msg).toBe("not found");
+      });
+  });
+});
 
 describe("/api/invalidPath", () => {
-    test("404: not found when querying a non-existent path", () => {
-      return request(app)
-        .get("/api/hajsdfbhjasdbfvja")
-        .expect(404)
-        .then(({ body }) => {
-          const { msg } = body;
-          expect(msg).toBe("Not found, invalid path.");
-        });
-    });
+  test("404: not found when querying a non-existent path", () => {
```

Comment:
You don't need to include this test on every end point. You only need to do this once and then you're 404 error handler will catch any paths that aren't specified